Meadowlark-js/tests/http/local.33b.http

```diff
@@ -5,21 +5,8 @@
 
 ##### START AUTHENTICATION SETUP
 
-### Bootstrap an admin
-# @name created_admin
-POST http://localhost:3000/local/oauth/client
-content-type: application/json
-
-{
-    "clientName": "Bootstrapped Admin",
-    "roles": [
-        "admin"
-    ]
-}
-
-###
-@admin_client_id={{created_admin.response.body.$.client_id}}
-@admin_client_secret={{created_admin.response.body.$.client_secret}}
+@admin_client_id=meadowlark_admin_key_1
+@admin_client_secret=meadowlark_admin_secret_1
```

Comment:
Restored the hard-coded credential in all *.http _except_ the authz file. Left the bootstrap admin process there, so that we can manually test/demonstrate that if desired.